src/inc-lib-init.php

```diff
@@ -1084,336 +1084,488 @@ function lovd_getInstallURL ($bFull = true)
+            '([-+]([0-9]+|\?))?' .               // 10. Latest intronic start position
+        '\))?)' .
+
+        '(_' .
+            '(\()?' .               // 13=(
+            '([-*]?[0-9]+|\?)' .                 // 14. (Earliest) end position
+            '([-+]([0-9]+|\?))?' .               // 15. (Earliest) intronic end position
+            '(?(13)_' . 
+                '([-*]?[0-9]+|\?)' .             // 17. Latest end position
+                '([-+]([0-9]+|\?))?' .           // 18. Latest intronic end position
+        '\)))?' .
+
+        '([ACGT]+>[ACGTRYSWKMBDHVN]+|' .         //  | (substitution)
+        '([ACTG]+\[[0-9]+])+' .                  //  | (repeat sequence)
+        '|ins|dup|delins|del|inv|[=?]' .         //  V
+        '|\|(gom|lom|met=))' .                   // 20. Type of variant
+
+        '(.*)))/',                               // 23. Suffix
+
+        $sVariant, $aMatches);
+
+    $aVariant = (!isset($aMatches[0]) ? [] : array(
+        'complete'                => $aMatches[0],
+        'prefix'                  => (!isset($aMatches[1])?  '' : $aMatches[1]),
+        'starting_parentheses'    => (!isset($aMatches[4])?  '' : $aMatches[4]), // The parentheses are given as a courtesy, to make additional checks easier.
+        'earliest_start'          => (!isset($aMatches[5])?   0 : $aMatches[5]), // These are not cast to integers, since they can still hold an informative '*'.
+        'earliest_intronic_start' => (!isset($aMatches[6])?   0 : (int)str_replace('?', '1', $aMatches[6])),
+        'latest_start'            => (!isset($aMatches[9])?   0 : $aMatches[9]),
+        'latest_intronic_start'   => (!isset($aMatches[10])?  0 : (int)str_replace('?', '1', $aMatches[10])),
+        'earliest_end'            => (!isset($aMatches[14])?  0 : $aMatches[14]),
+        'earliest_intronic_end'   => (!isset($aMatches[15])?  0 : (int)str_replace('?', '1', $aMatches[15])),
+        'latest_end'              => (!isset($aMatches[17])?  0 : $aMatches[17]),
+        'latest_intronic_end'     => (!isset($aMatches[18])?  0 : (int)str_replace('?', '1', $aMatches[18])),
+        'type'                    => (!isset($aMatches[20])? '' : $aMatches[20]),
+        'suffix'                  => (!isset($aMatches[23])? '' : $aMatches[23]),
+    ));
+
+    if (!isset($aVariant['complete']) || $aVariant['complete'] != $sVariant) {
+        // If the complete match from getHGVSpatternFromVariant is not set or does not equal the given variant,
+        //  the variant is not HGVS, and we cannot extract any information from it.
+        if ($bCheckHGVS) {
+            return false;
+        }
+        foreach (array('qter', 'pter', 'cen', '::', 'sup', 'bsr', 'rer') as $sUnsupported) {
+            if (strpos($sVariant, $sUnsupported)) {
+                $aResponse['errors']['ENOTSUPPORTED'] =
+                    'Currently, "' . $sUnsupported . '" is not yet supported by this HGVS check.';
```

Comment:
Dus `g.144329302|bsr` geeft geen positie terug?

Reply:
Klopt, die geeft geen positie terug

Reply:
Dat zou wel moeten, anders komen die helemaal vooraan te staan terwijl er duidelijk een positie in zit.
`pter`, `qter`, `cen`... daar kunnen we echt niets mee. `sup`, `bsr`, `rer` zijn dus, als ik Johan goed begrijp, gekoppeld aan `met=`, `lom`, en `gom`. Aangezien het niet officiëel is goedgekeurd, moeten we inderdaad een melding geven, maar de positie is er wel heel erg gemakkelijk uit te vissen.
Met andere woorden, deze varianten kunnen net zo worden afgehandeld zoals `c.1123T>Q`. Niet correct, maar we krijgen wel posities terug (toch?).